R/manual_layer_udf.R

```diff
@@ -255,6 +271,14 @@ execute_multi_array_udf <- function(namespace, array_list, udf=NULL, registered_
 
   # We rely on UDFLanguage$new to do enum-matching
   multi_array_udf$language <- UDFLanguage$new(language)
+  if (language == 'r') {
+    # For server-side stats; also for matching server-side Docker image version
+    # to client-side (us) version.
+    #
+    # TODO: How to specify lang vers for R calling registered Python UDFs.
+    # Probably make this a required argument (if language != 'r') once that's all deployed server-side.
+    multi_array_udf$version <- paste(R.version$major, R.version$minor, sep='.')
```

Comment:
It's the old 'if you need it three times maybe write a (local, unexported) helper function...)